.azure-pipelines/OfficialBuild.yml

```diff
@@ -63,16 +73,10 @@ steps:
 #     customCommand: run increment-version
 
 - task: Npm@1
-  displayName: 'Official Build and Package VSIX'
-  inputs:
-    command: custom
-    customCommand: run dist -- --feedPAT $(AZ_DevOps_Read_PAT) --isOfficialBuild true
-
-- task: Npm@1
-  displayName: 'Preview Build and Package VSIX'
+  displayName: 'Build and Package VSIX'
   inputs:
     command: custom
-    customCommand: run dist -- --feedPAT $(AZ_DevOps_Read_PAT) --isPreviewBuild true
+    customCommand: run dist -- --feedPAT $(AZ_DevOps_Read_PAT) --isOfficialBuild true --isPreviewBuild ${{ parameter.isPreRelease }}
```

Comment:
we just need to make sure that isPreviewBuild default to true in our pipelines to avoid future automatic release pipelines leaking out anything to stable version.

Reply:
The pre-release variable is set to true by default